docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -139,15 +138,19 @@ desired={<Five viewBox="140 40 680 500" />}
 
 <div>
 
-#### Current state
+| **Current State**                                                                                                                                   | **Desired State**                                                                                                                                                                                     |
+| --------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| The lost region has been ensured not to reconnect during the failover procedure. <br /> <br /> No data has been lost due to Zeebe data replication. | The lost brokers have been removed from the Zeebe cluster. <br /> <br /> Continued processing is enabled, and new brokers in the failback procedure will only join the cluster with our intervention. |
 
-You have previously ensured that the lost region cannot reconnect during the failover procedure.
+One of the regions is lost, meaning Zeebe:
 
-Due to the Zeebe data replication, no data has been lost.
+- Is unable to process new requests due to losing the quorum
+- Stops exporting new data to Elasticsearch in the lost region
+- Stops exporting new data to Elasticsearch in the survived region
 
-#### Desired state
+You have previously ensured that the lost region cannot reconnect during the failover procedure.
```

Comment:
same as above not applicable anymore.
We could rephrase it to something along the lines of
```
You have ensured that you fully lost a region and want to start the temporary recovery.
```

Reply:
actually copy / pasta as it's covered in the table above.